examples/aiohttp-echo/app.py

```diff
@@ -0,0 +1,105 @@
+# -*- coding: utf-8 -*-
```

Comment:
This is like trivia, not important. If this module is for Python 3 only, `coding: utf-8` doesn't need since [PEP 3120 -- Using UTF-8 as the default source encoding](https://www.python.org/dev/peps/pep-3120/) was defined on Python 3.

Reply:
Okay, I'll create another PR to remove this after merged this PR.